_config.yml

```diff
@@ -151,8 +151,9 @@ callouts:
 
 # Google Analytics Tracking (optional)
 # Supports a CSV of tracking ID strings (eg. "UA-1234567-89,G-1AB234CDE5")
-ga_tracking: UA-2709176-10,G-5FG1HLH3XQ
-ga_tracking_anonymize_ip: true # Use GDPR compliant Google Analytics settings (true/nil by default)
+# Note: the main Just the Docs site does *not* use Google Analytics.
+# ga_tracking: UA-2709176-10,G-5FG1HLH3XQ
+# ga_tracking_anonymize_ip: true # Use GDPR compliant Google Analytics settings (true/nil by default)
```

Comment:
The comment seems accurate. But couldn't it be misread as a claim that `ga_tracking_anonymize_ip: true` makes a JtD site automatically  GDPR-compliant?

Reply:
Hm. Agree it's misleading. Let me deal with this in the follow-up (where we also do the cookie consent banner, etc.).